docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -419,7 +419,24 @@ public interface ProtectedResourceService {
 ----
 
 `OidcClientRequestReactiveFilter` uses a default `OidcClient` by default. A named `OidcClient` can be selected with a `quarkus.oidc-client-reactive-filter.client-name` configuration property.
+You can also select `OidcClient` by setting `value` attribute of the `@OidcClientFilter` annotation. The client name set through annotation has higher priority than the `quarkus.oidc-client-reactive-filter.client-name` configuration property.
 
+[source,java]
```

Comment:
Please add `source:properties` just above it showing some default and jwt-secret definitions. Or may be a better idea, is to say something like "For example, given this `jwt-secret` named OIDC client declaration <shown in this section> (this is a link to the named oidc clients section) you can refer to this client like this:", something like that

Reply:
Done; I hope I didn't take "this" too literally, I wanted to avoid having `jwt-secret` inside link.